src/libraries/sendtohelix-mobile.targets

```diff
@@ -81,7 +81,7 @@
                                                                                  Destination="build/BuildIntegration" />
       <HelixCorrelationPayload Include="$(CoreCLRAotSdkDir)"                     Condition="'$(RuntimeFlavor)' == 'coreclr'"
                                                                                  Destination="build/aotsdk" />
-      <HelixCorrelationPayload Include="$(ToolsILLinkDir)"                       Destination="build/microsoft.net.illink.tasks" />
+      <HelixCorrelationPayload Include="$(ToolsILLinkDir)"                       Destination="build" />
```

Comment:
Is there a particular reason for changing that the ILLink tool lives under `build/microsoft.net.illink.tasks`?

Reply:
The path on Helix wasn't correct and probably not used. The ILLink is copied to the `<HelixCorrelationPayload Include="$(ToolsILLinkDir)" Destination="build" />`.